Docs: Update issue template. (#960)
zulip/zulip-desktop: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request
about: Suggest an idea for this project
---

**Problem Description**
<!-- Please add a clear and concise description of what the problem is. -->

> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

Reviewer: I think the comment and this statement is repetitive. What do you think? We should probably remove it.

Author: yes, thats what i was thinking. the heading  and comment is self explanatory. Ill remove the statement then.

> ** Proposed Solution**

Reviewer: ```suggestion
**Proposed Solution**
```

<!-- Describe the solution you'd like in a clear and concise manner -->

> A clear and concise description of what you want to happen.

Reviewer: Same as previous.

---
name: Feature request
about: Suggest an idea for this project

> title: ''

Reviewer: Same about titles, labels and assignees here.

labels: ''
assignees: ''

---

> **Is your feature request related to a problem? Please describe.**

Reviewer: ```suggestion
**Problem Description**
<!-- Please add a clear and concise description of what the problem is. -->
```

A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

> **Describe the solution you'd like**

Reviewer: ```suggestion
** Proposed Solution**
<!-- Describe the solution you'd like in a clear and concise manner -->
```

A clear and concise description of what you want to happen.

**Describe alternatives you've considered**

> A clear and concise description of any alternative solutions or features you've considered.

Reviewer: ```suggestion
<!-- A clear and concise description of any alternative solutions or features you've considered. -->
```

**Additional context**

> Add any other context or screenshots about the feature request here.

Reviewer: Put this also in a comment.
